### ISSUE TITLE
Page URL should contain at most 255 characters.

### ISSUE THREAD

**Issue by kippo** [verbb/formie#569]
**Description**

Forms with a hidden field with the default value of current URL throw the error `Page URL should contain at most 255 characters.` This limit is easily reached when the url contains a query string (for example one added from email marketing tools).

**Steps to reproduce**
1. Create a form with a hidden field with the default value of "Curren URL"
2. Submit the form on a page with a url and query string over 255 characters

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.4.15
- Craft version: 3.7.8
- Multi-site: Yes

**Comment by engram-design** [comment]
So there's probably two options here:

1. Provide "Current URL (with Query String)" and "Current URL (without Query String)" which would contain the full URL including the query string, and without (respectively)
2. Provide an option to set the content column size for a hidden field. It's set to 255 characters by default, but as you've demonstrated, this gets out of hand quickly.

Any preference on your end? Do you actually want to record the current URL with the query string?

**Comment by kippo** [comment]
Thanks for the quick response. Option 2 would be preferable as we can see a need to capture query strings.
That said both options would be ideal.

**Comment by engram-design** [comment]
Just added both options for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.15",
  "...": "..."
}
```

Then run `composer update`.

**Comment by kippo** [comment]
Lovely thank you.

**Comment by engram-design** [comment]
Fixed in [1.4.16](https://github.com/verbb/formie/releases/tag/1.4.16)
